spinalcordtoolbox/deepgmseg/deepgmseg.py

```diff
@@ -0,0 +1,215 @@
+# coding: utf-8
```

Comment:
What is this about? At least the arXiv link would be welcome...

Reply:
Same as above, which format are we using to add header docs ?